LayoutTests/accessibility/shadow-dom/reference-target/mac/aria-describedby.html

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Extra newline here I think, and in general there are a few extra newlines before and after a few tags (`html`, `head`, `body`, `</script>`) in this file that we don't typically go for in our tests.